deepfence_agent/Dockerfile

```diff
@@ -70,7 +70,7 @@ COPY --from=malware_build /usr/local/yara/lib /usr/local/lib
 
 RUN mkdir -p /home/deepfence/bin/yara-hunter \
     && apt-get update --allow-insecure-repositories \
-    && apt-get -qq -y --no-install-recommends install libjansson4 libssl1.1 libmagic1 bash curl python3-pip libhyperscan5 \
```

Comment:
Ideally we should remove all the `&&` and replace with separate `RUN`commands to benefit from docker build caching system, can we do that?